<a id="section-contect"></a>
[Jump to contect](#section-contect)


<H1>Q. 1 A car with a full tank of gas drives 80 miles per hour for 3 hours before running out of gas. How many miles can the car go per tank?<h1>

In [19]:
#80 miles is for one hours 
total_miles=80*3
print("Before running out of gas car can travel " + str(total_miles) + " miles")

Before running out of gas car can travel 240 miles


><h2> Q 2. A runner travels a half marathon (13.1 miles) in 3 hours. How fast was this runner going in miles per hour?<h2>

In [20]:
#13.1 miles in 3 hours 
miles_par_hours=13.1/3
print("Per hours Speed of runner is:" + str(miles_par_hours))

Per hours Speed of runner is:4.366666666666666


*<h3>Q 3. The spread of a virus is exponential. If 5 people are infected and each person infects 2 other people, how many people have been infected after 5 rounds of infection?<h3>*

In [23]:
infected_people = 5 * (2 ** 5)
print("total infected persons are:" + str(infected_person))

total infected persons are:160


here are the answares of the questions that we have to solve in this notebook 
- Before running out of gas car can travel 240 miles
- Per hours Speed of runner is:4.366666666666666
- total infected persons are:160



# Contect 

This is the contect detailes about me .



<h3> here is my github page have a look <h3>

[![here is my github page have look](https://logosmarcas.net/wp-content/uploads/2020/12/GitHub-Simbolo.png)](https://github.com/JaykishanJ)

<h3> here is my linkdin page have a look <h3>

[![here is my github page have look](https://www.onrec.com/sites/onrec/directory/files/Linkedin-Logo.png)](https://www.linkedin.com/in/jaykishan-solanki-7615b2146/)